libr/include/r_core.h

```diff
@@ -116,6 +116,11 @@ typedef struct r_core_undo_t {
 	ut64 offset;
 } RCoreUndo;
 
+typedef enum {
```

Comment:
remove this one?

Reply:
As pancake said, this is being used by ms autocomplete